Studio/src/Application/Optimize/OptimizeTool.cpp

```diff
@@ -4,23 +4,20 @@
 #include <QThread>
 #include <QMessageBox>
 #include <QTemporaryDir>
+#include <QIntValidator>
 
 #include <Optimize/OptimizeTool.h>
 #include <Libs/Optimize/OptimizeParameters.h>
 
 #include <Visualization/ShapeWorksWorker.h>
 #include <Data/Session.h>
-#include <Data/StudioMesh.h>
 #include <Data/Shape.h>
-#include <Data/StudioLog.h>
 
 #include <Optimize/QOptimize.h>
 
-#include <vtkPLYWriter.h>
-
 #include <ui_OptimizeTool.h>
 
-namespace shapeworks {
+using namespace shapeworks;
```

Comment:
just curious, is this convention preferred to the previous one?

Reply:
I don't really mind either way.  This does result in one less enclosing { brace.

It wasn't really an intentional change, I was having trouble with some type or something with a Qt slot.